tests/pw/utils/helpers.ts

```
@@ -466,6 +466,12 @@ export const helpers = {
         return `rgb(${r}, ${g}, ${b})`;
     },
 
+    // empty object values
+    emptyObjectValues: (obj: { [key: string]: any }) => (Object.keys(obj).forEach(key => (obj[key] = '')), obj),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)